src/components/SelectWithLogo/SelectWithLogo.tsx

```diff
@@ -0,0 +1,222 @@
+/*
+ *
+ *  * Copyright (c) 2022 AccelByte Inc. All Rights Reserved.
+ *  * This is licensed software from AccelByte Inc, for limitations
+ *  * and restrictions contact your company contract manager.
+ *
+ */
+
+import "simplebar/dist/simplebar.min.css";
+import * as React from "react";
+import Select, { ControlProps, MenuListComponentProps, OptionProps, SelectProps } from "@atlaskit/select";
+import { components, GroupTypeBase, SingleValueProps } from "react-select";
+import { default as classNames } from "classnames";
+import "./SelectWithLogo.scss";
+import { default as ReactTooltip } from "react-tooltip";
+import { getDomWidthByRef } from "../../utils/dom";
+import { translation } from "../../utils/i18n";
+import { SelectOption } from "../../types";
+
+import { default as SimpleBar } from "simplebar-react";
```

Comment:
actually i'm still curious why do we use simplebar here? doesn't select already limit its height and scroll if there are many options?

Reply:
the original component in AP used `simplebar` as well, so if I remove it then we will lose the cosmetic I guess, since all the scroll in sidebar are using `simplebar`